chain-signatures/node/src/protocol/signature.rs

```diff
@@ -835,9 +838,7 @@ impl SignGenerator {
                 Action::Wait => {
                     // Wait for the next set of messages to arrive.
                     let msg = self.recv().await.inspect_err(|_| {
-                        if self.proposer == me {
-                            signature_generator_failures_metric.inc();
-                        }
+                        signature_generator_failures_metric.inc();
```

Comment:
What is the reason for this change? The commit says that it want to include all failures in the metric now but now why.

Reply:
`node_account_id` is a stable identifier of the node provider. It never changes, unlike their setup (and instance), which may cause the loss of data integrity. But yes, we are wasting resources here. Mapping of instance -> node_account_id should be possible manually.

I will get back to this once we have request monitoring in place.

Reply:
@jakmeier  I've played with instances, since we are using K8s, the instance changes every time we reboot the node. Grafana recommends to use stable identifiers (node_account_id) In our case.